internal/cmd/commands/server/worker_initial_upstreams_reload_test.go

```diff
@@ -135,7 +135,6 @@ pollSecondController:
 		}
 	}
 
-	close(cmd.ShutdownCh)
-
+	cmd.ShutdownCh <- struct{}{}
```

Comment:
I think this is possibly taken care of by dev.go and server.go [using this](https://github.com/hashicorp/boundary/blob/e690ac6048d76f49a4c4729bb288d1e9157f3346/internal/cmd/base/base.go#L151-L160)

Tagging @jefferai for discussion since he suggested changing this to a counting chan